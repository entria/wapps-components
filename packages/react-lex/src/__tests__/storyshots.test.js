import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  storyKindRegex: /^react-lex|(Lex|Chat|Message)$/,
});
