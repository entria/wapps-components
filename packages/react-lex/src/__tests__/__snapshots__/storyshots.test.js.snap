// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots react-lex|Chat with Messages 1`] = `
<div
  className="sc-EHOje ikYPee"
>
  <div
    className="sc-bZQynM gGmxIh"
  >
    <div
      className="sc-htpNat cNiZJl"
    >
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Human!
          </p>
        </div>
      </div>
      <div>
        <div
          className="sc-bwzfXH ibBxMu"
          type="human"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello Bot!
          </p>
        </div>
      </div>
    </div>
    <div
      className="sc-bxivhb fZmUiV"
    >
      <form
        onSubmit={[Function]}
      >
        <input
          className="sc-ifAKCX cZUZpM"
          id="textInput"
          onChange={undefined}
          placeholder={undefined}
          type="text"
          value={undefined}
        />
      </form>
    </div>
  </div>
</div>
`;

exports[`Storyshots react-lex|Lex with BookTripBot 1`] = `
<div
  className="sc-EHOje ikYPee"
>
  <div
    className="sc-bZQynM gGmxIh"
  >
    <div
      className="sc-htpNat cNiZJl"
    >
      <div>
        <div
          className="sc-bwzfXH cTAZJH"
          type="bot"
        >
          <p
            className="sc-bdVaJa bcSwbK"
          >
            Hello, what can I help you with?
          </p>
        </div>
      </div>
    </div>
    <div
      className="sc-bxivhb fZmUiV"
    >
      <form
        onSubmit={[Function]}
      >
        <input
          className="sc-ifAKCX cZUZpM"
          id="textInput"
          onChange={[Function]}
          placeholder="Type here"
          type="text"
          value=""
        />
      </form>
    </div>
  </div>
</div>
`;

exports[`Storyshots react-lex|Message with botMessage 1`] = `
<div
  className="sc-bwzfXH cTAZJH"
  type="bot"
>
  <p
    className="sc-bdVaJa bcSwbK"
  >
    Hello Human!
  </p>
</div>
`;

exports[`Storyshots react-lex|Message with humanMessage 1`] = `
<div
  className="sc-bwzfXH ibBxMu"
  type="human"
>
  <p
    className="sc-bdVaJa bcSwbK"
  >
    Hello Bot!
  </p>
</div>
`;
